sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -467,6 +467,9 @@ class RapidsDriverPlugin extends DriverPlugin with Logging {
             conf.shuffleTransportEarlyStartHeartbeatInterval,
             conf.shuffleTransportEarlyStartHeartbeatTimeout)
       }
+    } else {
+      logWarning("Rapids shuffle is not available, configured shuffle manager is " +
```

Comment:
RAPIDS should always be capitalized as a standalone name.
```suggestion
      logWarning("RAPIDS shuffle is not available, configured shuffle manager is " +
```